saleor/graphql/order/mutations/orders.py

```diff
@@ -391,7 +392,8 @@ def perform_mutation(cls, _root, info, amount, **data):
         try_payment_action(
             order, info.context.user, payment, gateway.capture, payment, amount
         )
-
+        if order.is_fully_paid():
+            handle_fully_paid_order(order)
```

Comment:
This logic should go after `order_captured` call I guess? We first capture funds, then we're able to say that the order was fully paid. It's done like that in `order_created`.

Reply:
Yes, I wanted to ask about this because right now it works that way (there are even tests that are checking this exact order) and for me, it should be as you described.

Reply:
Fixed